apps/docs/content/components/forms/Form.mdx

```diff
@@ -1,49 +1,104 @@
 ---
 title: Form
-description: TBD
+description: Forms are commonly used to provide user interaction in web applications.
```

Comment:
Where does this description comes from? I'd rephrase it as it provides a generic description that could apply to button and other components.

This could be the description:
Forms are used to gather information from the user.

Reply:
this description comes from [Orbiter](https://wl-orbiter-website.netlify.app/?path=/docs/form--vertical-layout 😂